### DISCUSSION TITLE
Reviewing proposed changes in a pull request in local

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [community/community#34019] [Pull Requests]
There is a repo where I have no writing access. Others open a Pull Request for this repo, I want to review for it. But in Github website, it's not friendly for so many changes. I want to review it in my local machine. Is there best practice?

**Comment by 0xautohub** [comment]
Hey @awxiaoxian2020 Hope [this link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally) to help you.

**Reply by awxiaoxian2020** [reply]
I and the PRer fork the original repo, I can only checkout my repo PRs. But the PR is for original repo.

The main problem is that I have no write access for the original repo.

**Reply by airtower-luna** [reply]
You don't need write access to review changes. Fetch (read) the remote repository and compare the relevant branches (or other refs).

**Reply by awxiaoxian2020** [reply]
> You don't need write access to review changes. Fetch (read) the remote repository and compare the relevant branches (or other refs).

Yes，but what I want is a functional "code review" like Github website to contribute for original repo on existed PRs in local.
